ui/packages/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -121,6 +121,17 @@ module.exports = Marionette.ItemView.extend({
             Export Selected
           </div>
         </div>
+        <div
+          className={`query-interaction interaction-zipped-export ${exportClass} ${
+            disabledExport ? 'is-disabled' : ''
+          }`}
+          title="Export Selected (ZIP)"
+          data-help="Opens a form to export the selected search results in a compressed ZIP folder."
+        >
+          <div className={`interaction-text ${exportClass}`}>
+            Export Selected (ZIP)
```

Comment:
We could change the name of this option to be like Mac OS. Or simply `Compress Selected`. The inclusion of `(ZIP)` while specific feels technical and possibly negative to user experience. 

<img width="829" alt="Screen Shot 2019-03-15 at 12 00 08 PM" src="https://user-images.githubusercontent.com/7646577/54452096-e8fa8f00-4719-11e9-9276-ff24712d187b.png">

Reply:
Discussed with @TeresaHudson , we're going to move forward with the `Compressed` verbage.